src/igvfd/schemas/assay_term.json

```diff
@@ -33,12 +33,12 @@
         },
         "term_id": {
             "@type": "@id",
-            "title": "Ontology ID",
-            "description": "An ontology identifier describing an assay.",
+            "title": "Term ID",
+            "description": "A term identifier describing an assay.",
```

Comment:
i think the description should include indication that the term is ontological, rather than being vague.